### ISSUE TITLE
Return the error on shell session failure. Change console.log to msg emit

### ISSUE THREAD

**Issue by cmp-202** [cmp-202/ssh2shell#57]
From pull request #56 by eozer:

1. I have a rest api that uses ssh2shell, and if there are multiple concurrent requests ssh2shell is crashing. See the output below. Adding return statement on error resolves the problem.
So my setup:
• A collection of requests in postman that makes only GET requests to available routes
• More than 7 collection runner (sometimes it can handle even 10) at a time
• Request to the same IP
• 50 iterations for each collection runner

Server crashes with following:
```
Ready
192.168.1.120: Connection.error
192.168.1.120: Class.error
Error: MMI connection failed: 10010, Level: protocol
192.168.1.120: Connection.close
Closed
192.168.1.120: Class.error
Error: Unable to open shell, Level: undefined
192.168.1.120: Connection.shell
C:\code\restapi\node_modules\ssh2shell\lib\ssh2shell.js:807
_this._stream.setEncoding(_this.sshObj.streamEncoding);
^
TypeError: Cannot read property 'setEncoding' of undefined
at C:\code\restapi\node_modules\ssh2shell\lib\ssh2shell.js:807:27
at Array. (C:\code\restapi\node_modules\ssh2\lib\client.js:1362:14)
at SSH2Stream. (C:\code\restapi\node_modules\ssh2\lib\Channel.js:117:19)
at Object.onceWrapper (events.js:313:30)
at emitNone (events.js:106:13)
at SSH2Stream.emit (events.js:208:7)
at Socket. (C:\code\restapi\node_modules\ssh2\lib\client.js:331:24)
at emitOne (events.js:116:13)
at Socket.emit (events.js:211:7)
at TCP._handle.close [as _onclose] (net.js:557:12)
```
[nodemon] app crashed - waiting for file changes before starting...

2. Instead of writing the host output to the console, emit the message as usual. With this all ssh2shell logs covered by msg.send object.

*Moved this to an issue to make the changes to ssh2shell.coffee.*

Eozer identified the following corrections to resolve it.

* Change was need to remove an hard coded console.log() command when loading a new host when it should have been an .emit("msg", host); call. 

* If an error was received on shell ready the if statement didn't abort the reset of the shell callback function so it need a return command to bail at that point.

**Comment by cmp-202** [comment]
Published v1.9.1 
NextPrimaryHost debug returns the host ip of the before and next sshObj.server.host so no overload of information but helpful ip to identify respective host change.
Shell err detection now bails instead of keeping on going.

Its cool to see the multiple hosts functionality being used given that it has been there from pretty much the start. That console.log must have been a dirty debug I missed. Thanks for picking it up.

Pipe code is not working at the moment as it should so don't rely on it.

**Comment by cmp-202** [comment]
Is this issue able to be closed?
